pages/dashboards/index.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * @defgroup pages_submissions Submissions editorial page
+ */
+
+/**
+ * @file lib/pkp/pages/dashboards/index.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2003-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @ingroup pages_submissions
+ *
+ * @brief Handle requests for submissions functions.
+ *
+ */
+
+
+switch ($op) {
+    case 'index':
+        return new PKP\pages\dashboard\DashboardHandlerNext(PKP\pages\dashboard\DashboardPage::EDITORIAL_DASHBOARD);
```

Comment:
Have you considered not creating new pages for the dashboard-related requests (including `mySubmissions` and `reviewAssignments`)? Rather take a look at the `PKPBackendSubmissionsController` - I believe the best would be create new endpoints if needed. But I think we already have all necessary ones. Can you take a look at that?

Reply:
From frontend point of view I need to differentiate between editorial dashboard, my submissions and review assignment so user can switch between them. I think having different urls is the easiest to differentiate between what the user is after.

But maybe the urls structure could be   
```
.../dashboard/editorial
.../dashboard/my-submissions
.../dashboard/review-assignments
```

Instead of 
```
.../dashboards/
.../mySubmissions/
.../reviewAssignments/
```

What you think?

Reply:
PKPBackendSubmissionsController is driving responses from the _submissions endpoint and all these pages are using the same endpoint to get data. What I am trying to achieve here is really to be able clearly switch between 3 different 'dashboards' if the user has more roles.

Reply:
The `_submissions` endpoint is a structural part of that handler/controller. The request is handled by that handler depending on the part of the URL after that. What about endpoints we currently use? e.g., `_submissions/reviewAssignments` for the reviewer dashboard

Reply:
Ahh, sorry. We are talking about page handlers. In that case I'd use an operation (`op`) part of the URL to distinguish - the first structure that you mentioned. But let me think about this

Reply:
Yeah, it look to me like it's better to move that logic to the `DashboardHandlerNext`

Reply:
Alright.. will do that, its easy change. We might want to review at some point the set of urls we have to have some consistent naming. Because now there is also authorDashboard..   Naming is always hard. But given that these 3 things (editorial dashboard, my submission and review assignments) are very much the same screen with different data I like the idea of grouping that.